scripts/got_audit.py

```diff
@@ -27,9 +27,82 @@ class GotAuditCommand(GotCommand, GenericCommand):
     _cmdline_ = "got-audit"
     _syntax_ = f"{_cmdline_} [FUNCTION_NAME ...] "
     _example_ = "got-audit read printf exit"
-    _symbols_: Dict[str, List[str]] = collections.defaultdict(list)
-    _paths_: Dict[str, List[str]] = collections.defaultdict(list)
-    _expected_dups_ = ['__cxa_finalize']
+    _symbols_: dict[str, list[str]] = collections.defaultdict(list)
+    _paths_: dict[str, list[str]] = collections.defaultdict(list)
+
+    _expected_dups_ = [
```

Comment:
We might want to change this from a list to a set, since we are doing `short_name not in self._expected_dups_` that would require iterating the (now long) list each time.

Have you by chance done any profiling to see how this affects it? More of a curiosity. I think that in general a set is just a better idea.

Reply:
I have not profiled the difference, but using a set makes perfect sense to me.